vault-core/src/main/java/org/apache/jackrabbit/vault/fs/impl/io/JcrSysViewTransformer.java

```diff
@@ -129,6 +138,22 @@ public List<String> close() throws InvalidSerializedDataException {
             throw new InvalidSerializedDataException("Invalid sysview", e);
         }
 
+        // Apply previously skipped binary reference properties
+        for (Map.Entry<String, List<DocViewProperty2>> e : skippedProps.entrySet()) {
+            String path = e.getKey();
+            List<DocViewProperty2> props = e.getValue();
+            DocViewSAXHandler.log.trace("on '" + path + "' need to set binary reference properties: " + props);
+
+            try {
+                Node n = parent.getSession().getNode(path);
+                for (DocViewProperty2 prop : props) {
+                    prop.apply(n);
+                }
+            } catch (RepositoryException ex) {
+                throw new InvalidSerializedDataException("cannot apply reference properties to " + path, ex);
```

Comment:
reference property to me sound too much like property of type Reference. I would say:

"Cannot import binary reference properties in " (apply is hard to understand)